### DISCUSSION TITLE
我們是怎麼發現新行星的呢？

### DISCUSSION THREAD

**Discussion by alvin1226** [1101-GS3073-C/Python-Community-GS3073-C#684] [General]
對於太空系的我來說，系外行星一直是我們熱烈討論的話題之一，而當然科技發達的現在，我們也不用用肉眼來觀察了
利用機器學習讓AI學習"掩星法"來自主判斷是否有新行星出現，這可是讓科學家省了不少時間呢！
*掩星法就是觀察是否有物質從某個恆星前面通過，這會使該恆星的亮度降低，我們便可以從降低的光度來判斷這是不是一顆新行星
https://pansci.asia/archives/131876

**Comment by Valentinaa0214** [comment]
很棒的跨領域應用，利用機器學習來判斷新行星得出現，不只能減少人力的需求更能提高準確率，類似的技術一定能夠在未來對於人類探索宇宙有很大的幫助！

**Comment by Vincent20030314** [comment]
能夠減少人力需求也可以提高準確率，讓人很期待這項技術。

**Comment by HsuChen88** [comment]
行星會隨時間出現新的嗎?

**Comment by jerry0108566** [comment]
如果這項技術成熟的話也許可以結合在太空系所學做出一顆專門掩星觀測的立方衛星。
